app/lib/frontend/templates.dart

```diff
@@ -426,12 +428,15 @@ class TemplateService {
       links.add(<String, dynamic>{'href': href, 'label': label});
     }
 
-    if (repositoryUrl != homepageUrl) {
+    if (packageLinks.repositoryUrl != packageLinks.homepageUrl) {
       addLink(homepageUrl, 'Homepage');
     }
-    addLink(repositoryUrl, 'Repository', detectServiceProvider: true);
-    addLink(issueTrackerUrl, 'Issue Tracker', detectServiceProvider: true);
-    addLink(documentationUrl, 'Documentation');
+    addLink(packageLinks.repositoryUrl, 'Repository',
+        detectServiceProvider: true);
+    addLink(packageLinks.issueTrackerUrl, 'Issue Tracker');
```

Comment:
Shouldn't detectServiceProvider be true here?

Reply:
I thought that it is a bit boring to have "(GitHub)" label after each of links, and it is better to keep it only on the important one. The actual source code repository seems to be important, and also the report new issue, because the later requires an account on that site.